chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+    };
+    /**
+     * Repeat replace for max attempts or when done, whatever comes first
+     */
+    const repeatReplace = function (body, inRegexp, replaceWith) {
+      let replace1 = '';
+      for (let iter = max_replace_attempts_k; iter > 0; iter--) {
+        replace1 = body.replace(inRegexp, replaceWith);
+        if (body === replace1) {
+          iter = 0; // All done
+        } else {
+          body = replace1;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)